res/xml/preferences.xml

```diff
@@ -152,6 +152,13 @@
                     android:persistent="true"
                     android:defaultValue="true"/>
 
+            <CheckBoxPreference
+                    android:key="sync_read_flag"
+                    android:title="@string/ui_sync_read_flag_label"
+                    android:summary="@string/ui_sync_read_flag_desc"
+                    android:persistent="true"
+                    android:defaultValue="false"/>
+
```

Comment:
aren't the two options mutually exclusive ? if i set `sync_read_flag`, `mark_as_read` will be completely ignored.